src/lib/layer.js

```diff
@@ -47,6 +47,10 @@ const defaultProps = {
   onDragMove: noop,
   onDragEnd: noop,
   onDragCancel: noop,
+  // Offset depth based on layer index to avoid z-fighting.
+  // Negative values pull layer towards the camera
```

Comment:
I think the comment about direction could go into docs instead of/in addition to here.